magpie/ui/home/templates/error.mako

```diff
@@ -0,0 +1,44 @@
+<%inherit file="ui.home:templates/template.mako"/>
```

Comment:
Are we supposed to get a magpie themed error page? Because when I saw the 400 invalid post to remove the customizable flag of a group I rather saw a vanilla html error page.

Reply:
I've mostly tested for 401/403 use cases, but the intent is to wrap most of them. Will investigate.

Reply:
Should all be wrapped and correctly redirectly now with default error decorator.